### PR TITLE
Remove wrapper factory, move parameters resolver from `request-model`

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

- [x] Update readme

### CODE REVIEW

File: src/MiddlewareFactory.php [hunk 1]
```diff
@@ -121,4 +125,114 @@ private function isArrayDefinition(mixed $definition): bool
 
         return is_subclass_of((string) ($definition['class'] ?? ''), MiddlewareInterface::class);
     }
+
+    /**
+     * @param array{0:class-string, 1:string}|Closure $callable
+     */
+    private function wrap(array|Closure $callable): MiddlewareInterface
```
Comment (vjik) [inline src/MiddlewareFactory.php:132]:
```suggestion
    private function wrapCallable(array|callable $callable): MiddlewareInterface
```

Comment (vjik) [reply src/MiddlewareFactory.php:132]: Also need test with callable as array `[$object, 'method']`.

File: src/MiddlewareFactory.php [hunk 2]
```diff
@@ -121,4 +125,114 @@ private function isArrayDefinition(mixed $definition): bool
...
+
+    private function createCallableWrapper(callable $callback): MiddlewareInterface
+    {
+        return new class ($callback, $this->container, $this->parametersResolver) implements MiddlewareInterface {
+            private $callback;
+
+            public function __construct(
+                callable $callback,
+                private ContainerInterface $container,
+                private ParametersResolverInterface $parametersResolver
```
Comment (vjik) [inline src/MiddlewareFactory.php:149]: Maybe make parameter resolver is nullable. It's allow don't use refleaction for simple cases.

**Review by vjik — COMMENTED** [review]
Overall good. Fine idea!

File: src/MiddlewareFactory.php (hunk 1 above)
Comment (rustamwin) [reply src/MiddlewareFactory.php:132]: Currently does not support this.

File: src/MiddlewareFactory.php (hunk 2 above)
Comment (rustamwin) [reply src/MiddlewareFactory.php]: Done.

File: src/MiddlewareFactory.php (hunk 1 above)
Comment (vjik) [reply src/MiddlewareFactory.php]: Ok, I'm modified issue: https://github.com/yiisoft/middleware-dispatcher/issues/61

**Review by vjik — CHANGES_REQUESTED** [review]
Need fix psalm error and add line to changelog.

File: README.md [hunk 3]
```diff
@@ -96,28 +96,26 @@ executed first. For each middleware
 `\Yiisoft\Middleware\Dispatcher\Event\BeforeMiddleware` and `\Yiisoft\Middleware\Dispatcher\Event\AfterMiddleware`
 events are triggered.
 
-### Customizing callable wrapper
+### Creating your own implementation of parameters resolver.
```
Comment (samdark) [inline README.md:99]:
```suggestion
### Creating your own implementation of parameters resolver
```

**Review by samdark — APPROVED** [review]

**Review by vjik — APPROVED** [review]

**Pull request merged**